documentation/src/test/java/example/DynamicTestsDemo.java

```diff
@@ -41,6 +45,28 @@
 	}
 
 	@TestFactory
+	List<DynamicNode> dynamicTestsWithContainers() {
+		Executable alwaysTrue = () -> assertTrue(true);
+
+		List<DynamicNode> nestedOne = new ArrayList<>();
+		nestedOne.add(dynamicTest("nested-1", alwaysTrue));
+		nestedOne.add(dynamicTest("nested-2", alwaysTrue));
+		List<DynamicNode> nestedTwo = new ArrayList<>();
+		nestedTwo.add(dynamicTest("nested-3", alwaysTrue));
+		nestedTwo.add(dynamicContainer("2nd level", dynamicTestsFromIterable()));
+		nestedTwo.add(dynamicTest("nested-4", alwaysTrue));
+		nestedTwo.add(dynamicContainer("level II", dynamicTestsFromStream()));
+
+		List<DynamicNode> nodes = new ArrayList<>();
+		nodes.add(dynamicTest("begin", alwaysTrue));
+		nodes.add(dynamicContainer("container one", nestedOne));
+		nodes.add(dynamicTest("middle", alwaysTrue));
+		nodes.add(dynamicContainer("container two", nestedTwo));
+		nodes.add(dynamicTest("end", alwaysTrue));
+		return nodes;
+	}
+
+	@TestFactory
```

Comment:
I don't think this is the best example for the documentation. I'd rather show some real nesting. Thoughts?